src/unix/core.c

```diff
@@ -49,7 +49,7 @@
 # include <sys/wait.h>
 #endif
 
-#ifdef __APPLE__
+#if defined(__APPLE__) && !TARGET_OS_IPHONE
```

Comment:
This means ios won't use the O_CLOEXEC flag even though I'm 99% sure it supports that.

If it's just the use of `_NSGetEnviron()` that's problematic then you should probably break up the `#ifdef` block in two.

If it's the inclusion of `<mach-o/dyld.h>`, then you can just remove that header because I don't think it's actually used anymore. That said, `src/unix/darwin.c` uses it too to no ill effect.

Reply:
Sorry，I care about it. Thanks for your review!
Please help me review again！